operator/src/main/java/oracle/kubernetes/operator/DomainRecheck.java

```diff
@@ -164,11 +164,11 @@ public NextAction onSuccess(Packet packet, CallResponse<V1NamespaceList> callRes
       return doContinueListOrNext(callResponse, packet, createNextSteps(domainNamespaces));
     }
 
-    private Step createNextSteps(Set<String> namespacesToStartNow) {
+    private Step createNextSteps(Set<String> currentBatchOfNamespacesToStart) {
       List<Step> nextSteps = new ArrayList<>();
-      if (!namespacesToStartNow.isEmpty()) {
-        namespacesToStart.addAll(namespacesToStartNow);
-      }
+
+      // Add current batch of namespaces to start to the bigger list
```

Comment:
Looks better - this comment is not needed.

(most inline comments indicate that something isn't clear - your changes make the comment not needed)